cmake/compiler/arcmwdt/generic.cmake

```diff
@@ -29,12 +29,22 @@ endif()
 execute_process(
   COMMAND ${CMAKE_C_COMPILER} --version
   RESULT_VARIABLE ret
-  OUTPUT_QUIET
+  OUTPUT_VARIABLE full_version_output
   ERROR_QUIET
   )
 
 if(ret)
   message(FATAL_ERROR "Executing the below command failed. Are permissions set correctly?
   '${CMAKE_C_COMPILER} --version'"
   )
+else()
+  string(REGEX MATCH "[A-Z]-[0-9][0-9][0-9][0-9]\.[0-9][0-9]"
+     vers_string ${full_version_output})
+
+  string(REGEX MATCH "[0-9][0-9][0-9][0-9]\.[0-9][0-9]"
+     reduced_version ${vers_string})
+
+  if( ${reduced_version} VERSION_LESS "2022.06")
+    message(FATAL_ERROR "MWDT CCAC version must be not older than 2022.06")
```

Comment:
minor nit, when using `find_package()` in module mode to lookup a specific version, then the error message follows this structure:
```
Could NOT find <program>: Found unsuitable version "<version>", but required is at least "2022.06" (found <abs-path>/<program>)
```